src/pages/Login/Login.jsx

```diff
@@ -1,5 +1,211 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
 
 export default function Login() {
-  return <div></div>;
+  const [emailValue, setEmailValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const [isEmailValid, setIsEmailValid] = useState('');
+  const [isPasswordValid, setIsPasswordValid] = useState('');
+
+  const controlValue = event => {
+    if (event.target.name === 'inputEmail') {
+      setEmailValue(event.target.value);
+    } else if (event.target.name === 'inputPassword') {
+      setPasswordValue(event.target.value);
+    }
+  };
```

Comment:
```suggestion
  const controlValue = event => {
    const { name, value } = event.target; // 객체 구조 분해 할당

    if (name === 'inputEmail') setEmailValue(value);
    if (name === 'inputPassword') setPasswordValue(value);
  };
```
- 가독성을 위해 위와 같이 수정해볼 수도 있겠네요!

Reply:
넵 구조분해할당 하였습니다. else if 가 아니라 if 로 하여도 상관없네요 ! 그리고 굳이 안써도되는 중괄호 또한 생략했습니다